paddlenlp/prompt/prompt_model.py

```diff
@@ -120,3 +121,21 @@ def prompt_parameters(self):
         if self.verbalizer is not None:
             params += [p for p in self.verbalizer.parameters()]
         return params
+
+    def get_input_spec(self):
+        template_keywords = self.template.extract_template_keywords(self.template.prompt)
+        input_spec = [
+            InputSpec(shape=[None, None], dtype="int64"),  # input_ids,
+            InputSpec(shape=[None, None], dtype="int64"),  # token_type_ids
+            InputSpec(shape=[None, None], dtype="int64"),  # position_ids
+            InputSpec(shape=[None, None, None, None], dtype="float32"),  # attention_mask
+        ]
+        if "mask" in template_keywords:
+            input_spec.append(InputSpec(shape=[None], dtype="int64"))  # masked_positions
+        if "soft" in template_keywords:
+            if "mask" not in template_keywords:
+                input_spec.append(None)
+            input_spec.append(InputSpec(shape=[None, None], dtype="int64"))  # soft_token_ids
```

Comment:
Let's add some comments around here to explain why None is needed

Reply:
Ditto.